src/App.tsx

```diff
@@ -89,32 +63,29 @@ function App() {
     return (
         <Suspense fallback={<LoadingSpinner/>}>
             <QueryErrorResetBoundary>
-                {({ reset }) => {
-                    // eslint-disable-next-line @typescript-eslint/ban-ts-comment
-                    // @ts-ignore
```

Comment:
could you just leave a comment where you ignore ts issues? Better for other devs to see aswell

Reply:
Update `@types/react` and `@types/react-dom` to fix this ts error

Reply:
no I mean in the code itself
//Update @types/react and @types/react-dom to fix this ts error
//eslint-disable etc..
//@ts-ignore

Reply:
Yes, I understood you. I will leave it for the future